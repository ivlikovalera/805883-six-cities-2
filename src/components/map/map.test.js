import React from 'react';
import renderer from 'react-test-renderer';
import Map from './map.jsx';

it(`Map correctly renders after relaunch`, () => {
  const div = document.createElement(`div`);
  div.id = `map`; document.body.appendChild(div);
  const mapComponent = renderer
  .create(<Map
    pins={[]}
  />)
  .toJSON();

  expect(mapComponent).toMatchSnapshot();
});
