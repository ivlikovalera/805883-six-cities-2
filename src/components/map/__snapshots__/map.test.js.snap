// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Map correctly renders after relaunch 1`] = `
<div
  id="map"
  style={
    Object {
      "height": "100%",
      "width": "100%",
    }
  }
/>
`;
